bin/asc.js

```diff
@@ -803,15 +803,20 @@ function printStats(stats, output) {
 
 exports.printStats = printStats;
 
-var Buf = typeof global !== "undefined" && global.Buffer || Uint8Array;
+var allocBuffer = null;
+if (typeof global !== "undefined" && global.Buffer) {
+  allocBuffer = function (len) { return Buffer.alloc(len) };
```

Comment:
Could even use `allocUnsafe` here I guess, as the buffer is guaranteed to be completely filled where this function is used. Wdyt?

Reply:
Sure!